bazel/repository_locations.bzl

```diff
@@ -1112,13 +1112,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "VPP Comms Library",
         project_desc = "FD.io Vector Packet Processor (VPP) Comms Library",
         project_url = "https://fd.io/",
-        version = "7c3275e84b64ade4e20d00e4457bd4e437b1894f",
-        sha256 = "d456d37bbb7f90ec1ef166c1387e788b4c3078d38303f12ab41f1d0ac1a1cfc0",
+        version = "fe6d8a37062a28f28ee63c3e9177d18c10ed96e1",
+        sha256 = "2ae11c852f8c1967463251dcf9de983f9e4821abaeea6989fd2c1fb3020814b7",
```

Comment:
Can we use the release instead of a commit?
```
        version = "22.02",
        sha256 = "e488da74674ba26fabf17ac271e1208388220e275e7d9e5bc0dae226905a1d2d",
```

Reply:
Plan is to eventually get there but this commit includes some small VCL specific features and fixes. Unfortunately, because VCL is released together with VPP we don't have a way of updating/releasing them independently. 

Currently I try to limit updates to once per Envoy and VPP release, especially because testing suggests the integration is stable and performs pretty well, but we might still sporadically need some improvements and fixes.